keyboards/zeal60/keymaps/tusing/keymap.c

```diff
@@ -0,0 +1,50 @@
+// Default layout for Zeal60
+#include "zeal60.h"
+#include "action_layer.h"
+
+// For readability.
+#define _______ KC_TRNS
```

Comment:
This line isn't needed; `quantum_keycodes.h` (via `QMK_KEYBOARD_H`) defines this automatically.